browser/main/lib/ConfigManager.js

```diff
@@ -48,6 +48,7 @@ export const DEFAULT_CONFIG = {
     theme: 'default',
     showCopyNotification: true,
     disableDirectWrite: false,
+    scrollBar: true,
```

Comment:
@gregueiras just one small request. Can you change this to `showScrollBar` instead? Because other setting like `showCopyNotification` or `disableDirectWrite` are verb and I think this scroll bar setting should be a verb too. Thank you :+1:

Reply:
Yeah, sure, that wouldn't be a problem.

Nice to see you again 😄

Reply:
:smile: I was quite busy with school + other stuff. But I'll be able to have more time for this project now haha. Nice to see you too :+1: